packages/config/src/color.ts

```diff
@@ -11,13 +11,20 @@ export function parseHex(str: string): number {
   return val;
 }
 
+export interface RGBA {
```

Comment:
```suggestion
export interface Rgba {
```

We generally TitleCase everything